### DISCUSSION TITLE
Z123 workflow and results

### DISCUSSION THREAD

**Discussion by E2GO** [lllyasviel/stable-diffusion-webui-forge#54] [Q&A] [answered]
Hey! I'm getting strange result using z123 and can't find a model to check. It's not at outputs folder and results on screen. Any simple steps on workflow - appreciated.
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/1776669/cdb00dad-99c9-44e0-943a-9862a27b10f4)

**Comment by E2GO** [comment]
Tried different stuff today but still can't get it to work anyhow(((

**Comment by E2GO** [comment]
Anybody? So how to use it? There's no any output like 3d model but this strange pica at preview window... Also nothing created at output folder at all... 
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/1776669/09fed0b7-5b56-4a72-9aba-6219895c6157)

**Comment by strawberrymelonpanda** [comment, marked as answer]
I believe this implantation of Z123 doesn't make a 3d model, it makes a "novel view" based on the Elevation (vertical) and Azimuth (horizontal) settings. Think of it like moving the camera, and taking another picture of your subject.  You're getting 4 pictures because your batch size is set to 4.

As for the output location, open one of the results, right click it, and open it in a new tab.
That should tell you where the file is in the address bar.

**Reply by E2GO** [reply]
Oh! Now that what this about... Thanks for info. I misunderstood it then completely.

**Comment by strawberrymelonpanda** [comment]
> Oh! Now that what this about... Thanks for info. I misunderstood it then completely.

Maybe not entirely, just edited my answer to specify this implementation.

Reading [the repo](https://github.com/cvlab-columbia/zero123) again, it looks like it _can_ make 3d objects.  But I don't think this version is set up for it.  I think it does the top part, "Novel view synthesis".

**Comment by strawberrymelonpanda** [comment]
If you want to try 3d object generation, there's a few methods by the way.

Some instructions here:
https://huggingface.co/stabilityai/stable-zero123

> "To use Stable Zero123 for object 3D mesh generation in threestudio, you can follow these steps:"

**Comment by angrysky56** [comment]
Can we get this model useable please?
https://huggingface.co/stabilityai/stable-zero123

**Comment by strawberrymelonpanda** [comment]
It's working right now for me. I've been using it in the Z123 tab.  You can also see it in @E2GO 's screenshot.

**Reply by angrysky56** [reply]
My bad it's there now. I must have needed to restart not just refresh the folder search, thanks.

**Reply by angrysky56** [reply]
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/113643118/4a15ccb0-934a-4fd6-bdc7-92a2745b7db5)
It was a lq image to start.

**Comment by angrysky56** [comment]
![3dModelAlienExpRight](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/113643118/75c68c6c-9e42-4b90-91b0-3419180eb5c0)
![3dModelAlienExpLeft](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/113643118/aefe85c4-b5a9-458d-a299-0bf6d5d6d340)
![3dModelAlienExpFront](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/113643118/f018dbcd-b46b-46a1-b717-e9b0663b163b)

![3dModelAlienExpBack](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/113643118/12446119-1604-4779-8a79-887078459c81)
![3dModelAlienExpFrontBelow](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/113643118/45f1f29c-3ac7-4ca4-b71d-8b95ece2297e)
Someone should make an add on to let this make a model like ReActor does, but whole models you can have it make from standard angles of the subject and then add them in with something like RPG DiffusionMaster maybe move them around with OpenPose, also some kind of LoRA maker.
You will want to be able to add multiple models to an image or maybe even a LoRA and be able to specify the angle you want.

**Reply by MrMxyzptlk** [reply]
How did you get this output? I'm using sd-webui-forge...z123 extension...I have the correct ckpt...i have a simple no background image...no matter what I just get a picture that looks like a bunch of blobs. I'm running this on my 3060 GPU.

Thank you...
